### PR TITLE
Add some high-level documentation

### PR DESCRIPTION
**Pull request by brandonwillard**
Here's a start to some long overdue documentation for this project.

### CODE REVIEW

File: docs/src/symbolic-pymc-brief-tour.org [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+
+To "match" combinations of operations--or, in other words, graphs--like
+[[tf-print-graph]], we use *unification*.  Symbolic PyMC provides support for
+unification against TensorFlow graphs via *meta objects*.
+
+* Meta Objects
+
+Meta objects model the essential components of TensorFlow graphs, while allowing one
+to use input that isn't normally valid.  More specifically, we can
+construct meta graphs that contain *logic variables*.  Later, those logic
```
Comment (twiecki) [inline docs/src/symbolic-pymc-brief-tour.org]: Later occurs twice

**Comment by brandonwillard** [comment]
FYI: I've been updating the `gh-pages` branch, so the documentation in this PR is technically live: https://pymc-devs.github.io/symbolic-pymc/

**Comment by twiecki** [comment]
This looks great! What about the pymc4 example?

**Comment by brandonwillard** [comment]
> This looks great! What about the pymc4 example?

Good call.  I'll try to wrap that one up and put it on the site, too.

**Comment by brandonwillard** [comment]
FYI: I have to update the TensorFlow Radon example before I merge this.  Since I want to use the updated miniKanren relations (i.e. the ones currently in the documentation), the aforementioned updates depend on a fix I have for https://github.com/logpy/logpy/issues/81.

**Comment by brandonwillard** [comment]
I should add a [Beta-Binomial conjugate example](https://gist.github.com/brandonwillard/7c92ea0bb242a1abc3b3c6bd0f7c6f66) to the documentation, as well.

**Comment by twiecki** [comment]
That's a good one! What do you think of merging what's here already?

**Comment by brandonwillard** [comment]
@twiecki, yeah, I should merge this and just make separate PRs for new examples, improvements, etc.

**Pull request merged**